index.html

```diff
@@ -87,7 +87,7 @@ <h3>Marzoval</h3>
                         <iframe width="172" height="95" src="https://www.youtube.com/embed/5oZ9epHAdm4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                     </div>
                     <div class="content--text">
-                        <h3>The #1 Reason People Fail at Learning Web Development</h3>
+                        <h3>The Number 1 Reason People Fail at Learning Web Development</h3>
```

Comment:
Looks great but I think you should change Number 1 back to #1

Reply:
- [ ] Good job using inline comments [here](https://github.com/dselasea/project-review-trials/pull/1#discussion_r1015340630), but as requested by the previous reviewer 🗡️,  please always you should explain, "why", for each requested change. it helps the student to understand what to do 🍨.